ui/src/app/pages/profile/data-use-agreement-content.tsx

```diff
@@ -0,0 +1,200 @@
+            <br/>
+            If I become aware of any information that directly identifies one or more
+            participants, I will notify the <AoUTitle/> immediately
+            using the automatic notification system.
+          </IndentedListItem>
+          <IndentedListItem style={{margin: '0.5rem 0'}}>
+            I <strong>will <strong>NOT</strong> attempt</strong> to re-identify research
+            participants or their relatives.
+            <br/>
+            If I unintentionally re-identify participants through the process of my work, I will
+            contact the <AoUTitle/> immediately using the automatic
+            notification system.
+          </IndentedListItem>
+          <IndentedListItem style={{margin: '0.5rem 0'}}>
+            If I become aware of any uses or disclosures of <AoUTitle/> data that
+            could endanger the security or privacy of research participants, I will contact the
+            <AoUTitle/> immediately using the appropriate process.
+          </IndentedListItem>
+          <IndentedListItem>provide a meaningful description of my research purpose when establishing my <AoUTitle/>
+          workspace.</IndentedListItem>
+          <IndentedListItem>Within each workspace, I will use the data for the research purpose I have provided.
+          </IndentedListItem>
+          <IndentedListItem>This description will accurately reflect the demonstration project proposal for which I
+            received All of Us approval.</IndentedListItem>
+          <IndentedListItem>If I have a new research purpose, I will create a new workspace and provide a new
+            research purpose description.</IndentedListItem>
+          <IndentedListItem>Within my workspace, I will only use the data for the demonstration project for which I have
+            received All of Us approval.</IndentedListItem>
+          <IndentedListItem>use a version of the All of Us Research Program database that is current at or after the time my
+            analysis begins</IndentedListItem>
+          <IndentedListItem>take full responsibility for any external data, files, or software that I import into my
+            Workspace and the consequences thereof.</IndentedListItem>
+          <IndentedListItem>I will know and follow all applicable laws, regulations, and policies regarding access and use
+            for any external data, files, or software that I upload into my Workspace.</IndentedListItem>
+          <IndentedListItem>I will <strong>NOT</strong> upload data or files containing personally identifiable
+            information (PII).</IndentedListItem>
+          <IndentedListItem>PII means information that can be used to distinguish or trace the identity of an individual
+            (e.g., name, social security number, biometric records, etc.) either alone, or when combined with other
+            personal or identifying information that is linked or linkable to a specific individual (2 CFR §
```

Comment:
The citation should link to: https://www.govinfo.gov/content/pkg/CFR-2014-title2-vol1/pdf/CFR-2014-title2-vol1-sec200-79.pdf